### PR TITLE
pandas 1.0.5 -> pandas 1.1.0

### PR DESCRIPTION
**Pull request by itholic**
To match a latest pandas version for CI

스크린샷 2020-07-29 오후 3 05 32

### CODE REVIEW

**Comment by ueshin** [comment]
We shouldn't use RC versions. Let's mark this as `[WIP]` and wait for the official release.

**Comment by itholic** [comment]
@ueshin Oh, It means "Release Candidate". I didn't know that. Thanks! :)

**Comment by ueshin** [comment]
@itholic Seems like pandas 1.1.0 has been released. Could you update the PR to use pandas 1.1.0? Thanks!

**Comment by itholic** [comment]
Thanks for the notice, @ueshin !

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -102,15 +102,15 @@ jobs:
             pyarrow-version: 0.14.1
           - python-version: 3.7
             spark-version: 2.4.6
-            pandas-version: 1.0.5
+            pandas-version: 1.1.0
```
Comment (ueshin) [inline .github/workflows/master.yml:105]: As some of functions changed the behavior, shall we keep using `1.0.5` here?

Comment (itholic) [reply .github/workflows/master.yml]: Thanks, @ueshin .
Seems like test failure since the behavior has been changed.
I'll open the PR to fix them.

**Comment by itholic** [comment]
I close this and delegate to #1688

**Pull request closed without merging**